service/src/test/resources/uat/output/TC4-9465699918_Magre_prob1only_20210212.xml

```diff
@@ -54,10 +54,6 @@
         <ehrFolder classCode="FOLDER" moodCode="EVN">
             <id root="test-id-2" />
             <statusCode code="COMPLETE" />
-            <effectiveTime>
```

Comment:
I know these are effectively replaced in a future PR - but are there any valid cases where we don't output a ehrFolder.effectiveTime?